### PR TITLE
Feature/strmac 2399

### PR DESCRIPTION
**Pull request by sanelz**
- Extract `Plist2HTML` class to a common place

### CODE REVIEW

File: Strato/Plist2HTML.swift [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+enum Plist2HTML {
```
Comment (Tomusm) [inline Strato/Plist2HTML.swift:3]: I think we should rename it to something more precise, like `AcknowledgmentsPlistToHTML`, or `PodLicensesPlistToHTML`.

File: Strato/Plist2HTML.swift [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+	private static let blankLine = "<br>"
+	private static let ackKey = "PreferenceSpecifiers"
+	private static let titleKey = "Title"
+	private static let licenseKey = "License"
+	private static let footerKey = "FooterText"
+
+	/// Transforms the plist file containing the acknowledgments into an HTML
```
Comment (sirnacnud) [inline Strato/Plist2HTML.swift:11]:
```suggestion
	/// Transforms the plist file containing the acknowledgments into an HTML string
```

File: Strato/Plist2HTML.swift [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+enum Plist2HTML {
+
```
Comment (Tomusm) [inline Strato/Plist2HTML.swift:4]: Is there a reason why this is not in the HiDrive-iOS-Common-Framework?

Comment (sirnacnud) [reply Strato/Plist2HTML.swift:4]: I also thought this didn't belong in this submodule. I think we really should have some kind of framework which is shared app code between the iOS and mac app, as the Business framework and lower wouldn't use `Plist2HTML`.

File: Strato/Licences.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+require 'fileutils'
+
+class Licences
+
+	def self.generateCleanAcknowledgementsPlist(target, path)
```
Comment (Tomusm) [inline Strato/Licences.rb:5]: It's fine IMO 👍 
Can we just have a short documentation for the function explaining what each parameter is (at least their types) ?

File: Strato/Licences.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+require 'fileutils'
+
+class Licences
+
+	# target: pod target from the `Podfile`
+	# path: the files destionation path
```
Comment (sirnacnud) [inline Strato/Licences.rb:6]:
```suggestion
	# path: the file's destination path
```

**Review by Tomusm — APPROVED** [review]

**Review by sirnacnud — APPROVED** [review]

**Pull request merged**